aptos-move/block-executor/src/view.rs

```diff
@@ -134,12 +134,17 @@ impl<'a, T: Transaction, X: Executable> ParallelState<'a, T, X> {
                     .is_err()
                 {
                     // Inconsistency in recorded reads.
-                    Err(MVAggregatorsError::DeltaApplicationFailure)
+                    Err(PanicOr::Or(MVAggregatorsError::DeltaApplicationFailure))
```

Comment:
Seems like DeltaApplicationError isn't really right here?
It's only the case when we somehow called aggregated read twice and got different values, but (a) this should never happen if the extension works properly, and (b) it isn't about DeltaApplicationFailure?